contrib/masterfiles/git-failsafe.cf

```diff
@@ -198,7 +198,7 @@ bundle agent git_update
    "$(inputs_dir)/cf_promises_validated"
         comment => "Check whether a validation stamp is available for a new policy update to reduce the distributed load",
          handle => "check_valid_update",
-      copy_from => u_rcp("$(master_location)/cf_promises_validated","$(sys.policy_hub)"),
+      copy_from => u_rcp("$(master_location)/cf_promises_validated","$(sys.policy_hub_ip)"),
```

Comment:
Most `$(sys.policy_hub)` in the policy would be best to remain unchanged, since the code handles both IP and hostnames (especially true after the ACLs are fixed).

Just mentioning here to make sure we're on the same ship. :-)